scripts/install_cairo_bindings.sh

```diff
@@ -13,7 +13,6 @@ if [ "$1" == "--cleanup" ]; then
 fi
 
 function install_dev() {
-  git pull --recurse-submodules
```

Comment:
can you explain this change and how it is related?

Reply:
[Revert unwanted change, delete intermediate function and rename deplo…](https://github.com/software-mansion/protostar/pull/1845/commits/61d9b414d84f482849a114f029114024adb8c1ee)